composer.json

```diff
@@ -23,7 +23,7 @@
         "php": "^7.2",
         "symfony/framework-bundle": "~3.4|~4.0|~5.0",
         "doctrine/doctrine-bundle": "~1.0|~2.0",
-        "doctrine/migrations": "~3.0"
+        "doctrine/migrations": "^3.1@dev"
```

Comment:
This makes the PR a draft, right? We are not going to merge as is, are we?

Reply:
I not not see an issue with it. the `@dev` portion will be ignored by composer and for who requires this bundle, it will be seen as `^3.1`

Reply:
So… they won't be able to install it then?

Reply:
yes... ok, then we will wait for 3.1

Reply:
well, you can still merge the PR. But you need to release the library before release the bundle.

Reply:
@stof  exactly, that was my initial intention (cc @greg0ire )

Reply:
>  you can still merge the PR

That sounds dangerous :stuck_out_tongue:

Reply:
this has been solved, doctrine/migrations 3.1 has been released